src/sqlfluff/rules/L020.py

```diff
@@ -7,7 +7,29 @@
 
 
 class Rule_L020(BaseRule):
-    """Table aliases should be unique within each clause."""
+    """Table aliases should be unique within each clause.
+
+    | **Anti-pattern**
+    | In this example, same table 'foo' have two aliases 'f1' and 'f2'.
+
+    .. code-block:: sql
+
+        SELECT
+            f1.a,
+            f2.a
+        FROM foo f1, foo f2
+
+    | **Best practice**
+    | Make all table with the some aliases.
+
+    .. code-block:: sql
+
+        SELECT
+            f.a,
+            f.a
+        FROM foo f, foo f
+
+    """
```

Comment:
I think you've misunderstood the intent of this rule.

Would suggest this instead:

```suggestion
    """Table aliases should be unique within each clause.

    | **Anti-pattern**
    | In this example, the alias 't' is reused for two different ables:

    .. code-block:: sql

        SELECT
            t.a,
            t.b
        FROM foo AS t, bar AS t
        
        -- this can also happen when using schemas where the implicit alias is the table name:

        SELECT
            a,
            b
        FROM
            2020.foo,
            2021.foo

    | **Best practice**
    | Make all tables have a unique alias

    .. code-block:: sql

        SELECT
            f.a,
            b.b
        FROM foo AS f, bar AS b
        
        -- Also use explicit alias's when referencing two tables with same name from two different schemas

        SELECT
            f1.a,
            f2.b
        FROM
            2020.foo AS f1,
            2021.foo AS f2

    """
```

Reply:
Oh, I totally go to the way here, Thanks